yupana/processor/report_slice_processor.py

```diff
@@ -192,6 +192,38 @@ def generate_upload_candidates(self):
                       for key in host.keys() if key not in ['cause', 'status_code']}
         return candidates
 
+    def _remove_ip_addresses(self, host: dict):
```

Comment:
One can be confused by an existing method name as this method only removes `ip_addresses` key only when contains empty array or nothing.

```suggestion
    def _remove_empty_ip_addresses(self, host: dict):
```

Reply:
done.